src/app/containers/StoryPromo/index.jsx

```diff
@@ -82,7 +104,15 @@ const StoryPromo = ({ item, lazyLoadImage, topStory }) => {
       {headline && (
         <Headline script={script} service={service} topStory={topStory}>
           <Link href={url}>
-            <LinkContents item={item} />
+            {isLive ? (
+              <LiveComponent
+                service={service}
+                headline={linkcontents}
```

Comment:
Is this right? I meant `headline={headline}`

Reply:
`const linkcontents = <LinkContents item={item} />; `

Reply:
Chatted IRL, using `linkcontents`, not `headline` is to make use of `overtyped || headline` in `linkcontents` :)